src/defaultMonster.js

```diff
@@ -34,6 +34,7 @@ import { ModernMT } from './translators/modernMT.js';
 import { DeepL } from './translators/deepL.js';
 import { GoogleCloudTranslateV3 } from './translators/gctV3.js';
 import * as regexNormalizers from './normalizers/regex.js';
+import * as keywordTranslatorMaker from './normalizers/keywordTranslatorMaker.js';
```

Comment:
This is weird. The import makes it sounds like multiple normalizers may be imported but the name makes it sound like only one will. Either change the name if you think that file will eventually export more than one entry, or change the import to expect exactly one.

Reply:
Changed to import exactly one.